README.md

```diff
@@ -92,11 +92,40 @@ The argument command that got cut off there is `jupyter_ascending.requests.execu
 You'll have to set up a keyboard shortcut for this external tool in the keyboard shortcuts menu.
 
 
-
 ## Usage in Vim
 
 To use in vim, see: [jupyter_ascending.vim](https://github.com/untitled-ai/jupyter_ascending.vim)
 
+## Usage in Visual Studio Code
+
+(this will only work if you have a project folder open - it won't work if you just have an individual file open)
```

Comment:
This note seems weird here, maybe turn this into a sentence and remove the parentheses?